src/privacy_review_assistant_group.py

```diff
@@ -5,25 +5,28 @@
 from rag_agents import setup_rag_assistant
 from threat_model_reviewer_group import ThreatModelReviewerGroup
 from threat_model_visualizer import ThreatModelImageVisualizerCapability
+from xml_threat_model_reviewer import setup_xml_threat_model_reviewer
+
+USE_XML_ASSISTANT=True
```

Comment:
You can put this in the state of the conversation too, or an environment variable.

If you'll do it in state, you can add a new variable in AppConversationState. Then copy something like https://github.com/heyitsaamir/autogen-privacy-helper/blob/17cb0f5f736837f5250260d229d0f8fe5e64f30c/src/bot.py#L82-L87 to update this value in the state. So then you can do `/useXML` from teams that'll start using the xml value.

Reply:
(or vice versa)

Reply:
I'll add this as a follow up